### ISSUE TITLE
Studio can't access source maps with Firefox as login cookies are not sent when requesting source maps

### ISSUE THREAD

**Issue by LukasKalbertodt** [opencast/opencast#1495]
When opening Studio in **Opencast 8.3** in Firefox, the following error appears in the dev console:

```
Source map error: Error: JSON.parse: unexpected character at line 1 column 1 of the JSON data
Resource URL: https://stable.opencast.org/studio/static/js/main.2002447c.chunk.js
Source Map URL: main.2002447c.chunk.js.map
```

The reason for this is that when Firefox sends the HTTP request for those `.map` files, login cookies are not included. This in turn means that the request is not allowed and gets redirected to the login page. The login page HTML is not valid JSON (duh), so we get the error above. Interestingly, Chrome includes cookies in the requests for source map files. 

(Please note that you won't be able to see the HTTP requests for those map files in the network tab of Firefox/Chrome dev tools. I used Wireshark to inspect the requests.)

I don't think this bug can be fixed by Studio, so I created this issue here. I think one solution would be to add this line to `mh_default_org.xml`:

```xml
    <sec:intercept-url pattern="/studio/static/js/**" access="ROLE_ANONYMOUS" />
```

I don't think making that URL publicly accessible is a problem. That folder just contains the JS, `.map` and two license files.

**Comment by lkiesow** [comment]
You could have just created a pull request 😸

**Comment by LukasKalbertodt** [comment]
Fixed by #1509
